provided additional info on snmpv3 (#3570)
SkylineCommunications/dataminer-docs: develop/devguide/Connector/ConnectionsSnmpSnmpv3.md

- Authentication algorithms:
  - MD5
  - SHA128
  - SHA224 (Since DataMiner 9.6.13 (RN 23586))
  - SHA256 (Since DataMiner 9.6.13 (RN 23586))
  - SHA384 (Since DataMiner 9.6.13 (RN 23586))
  - SHA512 (Since DataMiner 9.6.13 (RN 23586))

> [!NOTE]
> When an encryption algorithm is used with more bits than the authentication type, this can cause issues. The following combinations are therefore not possible:
>
> - MD5 & AES192
> - MD5 & AES256
> - SHA128 & AES192
> - SHA128 & AES256
> - SHA224 & AES256

## About snmpEngineID, snmpEngineBoots, snmpEngineTime and timeliness

[RFC 3411](https://datatracker.ietf.org/doc/html/rfc3411#section-3.1.1.1) states the following:

*Within an administrative domain, an snmpEngineID is the unique and unambiguous identifier of an SNMP engine. Since there is a one-to-one association between SNMP engines and SNMP entities, it also uniquely and unambiguously identifies the SNMP entity within that administrative domain. Note that it is possible for SNMP entities in different administrative domains to have the same value for snmpEngineID. Federation of administrative domains may necessitate assignment of new values.*

> Therefore, if DataMiner manages multiple SNMP entities, it is important that these all configured with a different value for *snmpEngineID*.

Reviewer: ```suggestion
Therefore, if DataMiner manages multiple SNMP entities, it is important that these are all configured with a different value for *snmpEngineID*.
```

If this is not the case, this can lead to communication issues:

Every SNMP engine maintains two values:

> - *snmpEngineBoots*: a count of the number of times the SNMP engine has re-booted/re-initialized since *snmpEngineID* was last configured

Reviewer: ```suggestion
- *snmpEngineBoots*: a count of the number of times the SNMP engine has rebooted/reinitialized since *snmpEngineID* was last configured
```

These values are included in an authenticated message sent to or received from that SNMP engine (a non-authoritative SNMP engine retrieves these values from an authoritative SNMP engine by a [discovery process](https://datatracker.ietf.org/doc/html/rfc3414#section-4)). This is because when an SNMP engine receives an authenticated message, it verifies whether the time indication in the message falls within a specific time window compared to its current time. This acts as a message replay protection mechanism (For more information, refer to [RFC3414](https://datatracker.ietf.org/doc/html/rfc3414#section-1.5.2)).

[RFC 3414](https://datatracker.ietf.org/doc/html/rfc3414#section-2.3) also states the following:

*A non-authoritative SNMP engine must keep local notions of these values (snmpEngineBoots, snmpEngineTime and latestReceivedEngineTime) for each authoritative SNMP engine with which it wishes to communicate.  Since each authoritative SNMP engine is uniquely and unambiguously identified by its value of snmpEngineID, the non-authoritative SNMP engine may use this value as a key in order to cache its local notions of these values.*

> If you would have multiple devices with the same *snmpEngineID*, this could result in the the non-authoritative SNMP engine to only be able to keep track of the notion of time of one authoritative SNMP engine (which could lead to communication issues with all SNMP engines that have the same value for *snmpEngineID*).

Reviewer: ```suggestion
If you would have multiple devices with the same *snmpEngineID*, this could result in the non-authoritative SNMP engine to only be able to keep track of the notion of time of one authoritative SNMP engine (which could lead to communication issues with all SNMP engines that have the same value for *snmpEngineID*).
```
